src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg

```diff
@@ -129,7 +134,17 @@
             "wrap": true,
             "size": "Large"
         },
-        ${EmployeeList(employees)}
+        ${EmployeeList(employees)},
+        {
+            "type": "ActionSet",
+            "actions": [
+                {
+                    "type": "Action.OpenUrl",
+                    "title": "More booking info",
+                    "url": "https://app.powerbi.com/groups/me/reports/a4069dc4-86c1-4cba-bf81-161bb108c5c2/ReportSection"

```

Comment:
Please put the url in appsettings.json, and reference it using `settings` variable